### DISCUSSION TITLE
Ability to insert a folded block at the end of the quote

### DISCUSSION THREAD

**Discussion by monryse** [sundevista/local-quotes#28] [Ideas]
I love so much this plug-in, finally in obsidian i can randomly insert some quotation collected by me in my daily notes.

Quotes collected by me are made of a very short quote and a little excerpt, so the whole block included in the not will not take to much space, like this:

![image](https://github.com/sundevista/local-quotes/assets/137269/acc51ef5-7d84-4e71-90d9-c37821e2569e)

Sometime, there are quotes that deserves a deep understanding, so i thought about an expandable block, at the end of the quote, that will show more text. The block should be folded by default.

Kind regards, 
Bart.

**Comment by sundevista** [comment]
Hi, Bart! This idea, unfortunately, requires changes to quote vault structure and comes with a lot of API changes, so I can't afford myself to do it because of lack of time. But I can suggest you add a link at the end of the quote with an explanation you need. I won't delete the discussion and maybe would add the feature if I have time. Thank you for using Local Quotes!
